### PR TITLE
Fix Spawn PlanetRuin

### PR DESCRIPTION
**Pull request by ZephyrTFA**
Fixes errors with Spawn Planet/Ruin not functioning after latest O4 PR

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -817,54 +817,46 @@
...
-				select_from = SSmapping.yellow_ruins_templates
-			if("Everything")
-				select_from = SSmapping.ruins_templates
-			else
-				select_from = null
+		return
+
+	var/ruintype = tgui_input_list(usr, "What type of ruin?", "Spawn Ruin", RUINTYPE_LIST_ALL, 60 SECONDS)
+	if(!ruintype)
+		if(tgui_alert(usr, "Did you mean to not have a ruin?", "Spawn Planet/Ruin", list("Yes", "No"), 10) != "Yes")
```
Comment (Fikou) [inline code/modules/admin/verbs/randomverbs.dm]: what is 10 here

Comment (ZephyrTFA) [reply code/modules/admin/verbs/randomverbs.dm]: That's supposed to be 10 SECONDS

Comment (ZephyrTFA) [reply code/modules/admin/verbs/randomverbs.dm]:
```suggestion
		if(tgui_alert(usr, "Did you mean to not have a ruin?", "Spawn Planet/Ruin", list("Yes", "No"), 10 SECONDS) != "Yes")
```

**Comment by MarkSuckerberg** [comment]
I really should have datumised planet types

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by Latentish** [comment]
Hit this with the gas please.
And tm it please, even if it doesn't work we can check it.

**Comment by ZephyrTFA** [comment]
Fuck it, I'm self merging this and you literally cannot stop me

**Pull request merged**